src/datahike/query.cljc

```diff
@@ -178,37 +181,39 @@
       (aset res (+ l1 i) (#?(:cljs da/aget :clj get) t2 (aget idxs2 i)))) ;; FIXME aget
     res))
 
-(defn sum-rel [a b]
-  (let [{attrs-a :attrs, tuples-a :tuples} a
-        {attrs-b :attrs, tuples-b :tuples} b]
-    (cond
-      (= attrs-a attrs-b)
-      (Relation. attrs-a (into (vec tuples-a) tuples-b))
-
-      (not (same-keys? attrs-a attrs-b))
-      (dt/raise "Can't sum relations with different attrs: " attrs-a " and " attrs-b
-                {:error :query/where})
-
-      (every? number? (vals attrs-a))                       ;; can’t conj into BTSetIter
-      (let [idxb->idxa (vec (for [[sym idx-b] attrs-b]
-                              [idx-b (attrs-a sym)]))
-            tlen (->> (vals attrs-a) (reduce max) (inc))
-            tuples' (persistent!
-                     (reduce
-                      (fn [acc tuple-b]
-                        (let [tuple' (da/make-array tlen)]
-                          (doseq [[idx-b idx-a] idxb->idxa]
-                            (aset tuple' idx-a (#?(:cljs da/aget :clj get) tuple-b idx-b)))
-                          (conj! acc tuple')))
-                      (transient (vec tuples-a))
-                      tuples-b))]
-        (Relation. attrs-a tuples'))
-
-      :else
-      (let [all-attrs (zipmap (keys (merge attrs-a attrs-b)) (range))]
-        (-> (Relation. all-attrs [])
-            (sum-rel a)
-            (sum-rel b))))))
+(defn sum-rel
+  ([a] a)
```

Comment:
Why is this needed now?

Reply:
Good point! I don't think we need it.

I believe the reason why it was there is probably that I wanted to use `sum-rel` as a step function for a transducer and that would require this arity. But I probably dropped that approach as it would have made this PR less focused.